src/tests/index.js

```diff
@@ -142,4 +142,34 @@ describe('<FuzzySearch />', () => {
     // Each result should have a 'matches' array now with `includeMatches`
     expect(wrapper.state('results')[0].item.title).to.equal('The Great Gatsby');
   });
+
+  it('should support style props', () => {
+     const onChange = sinon.spy();
+     const wrapper = mount(
+      <FuzzySearch
+        keyForDisplayName="author"
+        list={list}
+        onSelect={onChange}
+        keys={['author', 'title']}
+        options={{ includeMatches: true }}
+        inputStyle={{
+          outline: '1px solid red',
+        }}
+        inputWrapperStyle={{
+          outline: '1px solid red',
+        }}
+        resultStyle={{
+          backgroundColor: 'yellow',
+        }}
+        resultsWrapperStyle={{
+          border: '2px solid blue',
+        }}
+        selectedResultStyle={{
+          backgroundColor: 'green',
+        }}
+      />,
+    );
+
+    expect(wrapper.find('input')).to.exist;
```

Comment:
Could use some help making this assertion more helpful.. Was having trouble checking for style props.

Reply:
I think we should not invest time here and get rid of enzyme altogether since it's no longer maintained. We can use react-testing-library as a replacement.